test/e2e/operator_test.go

```diff
@@ -415,8 +426,10 @@ func TestDNSForwarding(t *testing.T) {
 		}
 	}()
 	// Wait for the client Pod to be ready.
+	name = types.NamespacedName{Namespace: testClient.Namespace, Name: testClient.Name}
 	err = wait.PollImmediate(1*time.Second, 60*time.Second, func() (bool, error) {
-		if err := cl.Get(context.TODO(), types.NamespacedName{Namespace: testClient.Namespace, Name: testClient.Name}, testClient); err != nil {
+		if err := cl.Get(context.TODO(), name, testClient); err != nil {
+			t.Logf("failed to get pod %s: %v", name, err)
```

Comment:
The log msg will look better with:
`t.Logf("failed to get deployment %s/%s: %v", name.Namespace, name.Name, err)`